ci-operator/step-registry/cucushift/upgrade/check/capability/cucushift-upgrade-check-capability-commands.sh

```diff
@@ -329,9 +336,28 @@ if [[ "${baselinecaps_from_cluster}" == "v4."* ]]; then
     else
         echo "migrated_caps is not empty."
         for key in "${!migrated_caps[@]}"; do
-            if [[ $source_enabled_caps == *$key* ]] && [[ $target_enabled_caps != *${migrated_caps[$key]}* ]]; then
-                expected_enabled_caps="${expected_enabled_caps} ${migrated_caps[$key]}"
-                expected_implicit_caps="${expected_implicit_caps} ${migrated_caps[$key]}"
+            # Handle the scenario of multiple caps in key
+            IFS='+' read -r -a key_caps <<< "$key"
+            all_key_caps_exist=true
+            for cap in "${key_caps[@]}"; do
+                if [[ ! " ${source_enabled_caps} " =~ ${cap} ]]; then
```

Comment:
```suggestion
                if [[ ! " ${source_enabled_caps} " =~ " ${cap} " ]]; then
```
Here is my local testing:
```
$ source_enabled_caps="ab"
$ cap="a"
$ if [[ ! " ${source_enabled_caps} " =~ ${cap} ]]; then echo "no have it"; fi
$ if [[ ! " ${source_enabled_caps} " =~ ${cap} ]]; then echo "no have it"; else echo "have it"; fi
have it
$ if [[ ! " ${source_enabled_caps} " =~ " ${cap} " ]]; then echo "no have it"; else echo "have it"; fi
no have it
```

Reply:
hmm, I did quote that, but [shellcheck](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/53222/pull-ci-openshift-release-master-step-registry-shellcheck/1802592763439484928) suggested to drop it.

```
In .//ci-operator/step-registry/cucushift/upgrade/check/capability/cucushift-upgrade-check-capability-commands.sh line 353:
                if [[ ! " ${target_enabled_caps} " =~ " ${cap} " ]]; then
                                                      ^--------^ SC2076: Remove quotes from right-hand side of =~ to match as a regex rather than literally.
```

Reply:
We can insert a comment line - `#shellcheck disable=SC2076` to disable the warning.

Reply:
Makes sense. Updated!